### ISSUE TITLE
what should our signature names look like for databases?

### ISSUE THREAD

**Issue by ctb** [dib-lab/sourmash_databases#9]
for genbank, we do `--name-from-first`, so we get output like this:

`CP001941.1 Aciduliprofundum boonei T4...`

for gtdb, we do trickier name setting, so we get output like this:

`GCF_000025665 s__Aciduliprofundum boonei`

with the main difference here being that the GCF_ identifier points to the identifier for the whole genome, not just the first sequence. That seems better.

We _could_ add an optional identifier string to signatures. Hrm. Ref https://github.com/dib-lab/sourmash/issues/268 for more such questions.

ref #7

**Comment by luizirber** [comment]
For `wort` I generated a name closer to GTBD:
`GCF_000246355.1 Leptospira kirschneri serovar Mozdok str. 'B 81/7 type 3/Tsaratsovo' strain=B 81/7 type 3/Tsaratsovo, CLC_glsol0`

I generate it from `assembly_summary.txt`, using `assembly_accession`, `organism_name`, `infraspecific_name` (if there is one) and finally a comma and `asm_name`.

This example is pretty much the worst case I found: long name, with `'` in the middle (so I need to escape properly in the shell). But the crucial point is using `GCF_000246355.1` in the first position, because `--name-from-first` in NCBI assemblies is a mess for our use cases.
